package/crds/acm.aws.crossplane.io_certificates.yaml

```diff
@@ -678,6 +678,31 @@ spec:
                       - validationDomain
                       type: object
                     type: array
+                  keyAlgorithm:
+                    description: 'Specifies the algorithm of the public and private
+                      key pair that your certificate uses to encrypt data. RSA is
+                      the default key algorithm for ACM certificates. Elliptic Curve
+                      Digital Signature Algorithm (ECDSA) keys are smaller, offering
+                      security comparable to RSA keys but with greater computing efficiency.
+                      However, ECDSA is not supported by all network clients. Some
+                      AWS services may require RSA keys, or only support ECDSA keys
+                      of a particular size, while others allow the use of either RSA
+                      and ECDSA keys to ensure that compatibility is not broken. Check
+                      the requirements for the AWS service where you plan to deploy
+                      your certificate. Default: RSA_2048 Note: ACM can request the
+                      issue of new certificates using the following algorithms: "RSA_2048",
+                      "EC_prime256v1" and "EC_secp384r1". The remaining algorithms
+                      are supported only for imported certificates. See also AWS docs:
+                      https://docs.aws.amazon.com/acm/latest/userguide/acm-certificate.html#algorithms.title'
+                    enum:
+                    - RSA_1024
+                    - RSA_2048
+                    - RSA_3072
+                    - RSA_4096
+                    - EC_prime256v1
+                    - EC_secp384r1
+                    - EC_secp521r1
```

Comment:
Just a thought, but do we want to create an enum here?
If AWS ever adds new values, we will need to adjust this here. 

Maybe it makes more sense to just add it to the description?

Reply:
Removed enums. Should be enough information n the description.
Also there will be an AWS Error if the string in `KeyAlgorithm` is not valid